src/components/DeleteConfirmationModal.vue

```diff
@@ -1,13 +1,9 @@
-<template functional>
```

Comment:
Note for future readers: we could have make the FAIcon component function (as the FontAwesomeIcon is functional) but we decided not ot use them as they will be no more in vue 3 https://v3.vuejs.org/guide/migration/functional-components.html